app/src/main/cpp/skyline/services/audio/IAudioRenderer/voice.cpp

```diff
@@ -60,6 +67,13 @@ namespace skyline::service::audio::IAudioRenderer {
                 samples.resize(currentBuffer.size / sizeof(i16));
                 state.process->ReadMemory(samples.data(), currentBuffer.address, currentBuffer.size);
                 break;
+            case skyline::audio::AudioFormat::ADPCM:
+                {
```

Comment:
The bracket for a case statement should be on the same line at the end of the case statement itself

Reply:
Okay, do I end it the same way?

Reply:
The bracket at the end can be on it's own line, it's just that the starting one has to share the line with the rest of the statement, just like we have it for `if`/`for`/`while` and other bracketed control flow statements.